src/blockdata/block.rs

```diff
@@ -411,7 +424,10 @@ mod tests {
         assert_eq!(real_decode.header.bits, 486604799);
         assert_eq!(real_decode.header.nonce, 2067413810);
         assert_eq!(real_decode.header.work(), work);
-        assert_eq!(real_decode.header.validate_pow(&real_decode.header.target()).unwrap(), real_decode.block_hash());
+        assert_eq!(
+            real_decode.header.validate_pow(&real_decode.header.target()).unwrap(),
+            real_decode.block_hash()
+        );
```

Comment:
oh, this one is really subjective then. For my eyes this is the best formatting for a function call that puts its first argument on a newline.

Reply:
I also prefer the rustfmt formatting here, it makes it clear where the call to `assert_eq` ends. Otherwise it's like reading Python where half the program semantics are invisible.